index.js

```diff
@@ -77,7 +77,10 @@ module.exports = {
 
   testemMiddleware: function(app) {
     if (!this._isCoverageEnabled()) { return; }
-    attachMiddleware(app, { root: this.project.root, ui: this.ui });
+    attachMiddleware(app, {
+      root: this.project.root,
+      configDirName: path.dirname(this.project.configPath())
```

Comment:
1. I removed `ui` because it was not being reference from the middleware so the parameter was unnecessary. 

2. I changed the config code to now take in the full `project` and construct the path inside.